ui/apps/platform/src/Containers/SystemConfig/Details/PlatformComponentsConfigDetails.tsx

```diff
@@ -0,0 +1,152 @@
+            },
+        },
+        {
+            name: 'custom platform component 2',
+            namespaceRule: {
+                regex: '^nvidia$',
+            },
+        },
+    ],
+};
+
+const PlatformComponentsConfigDetails = (): ReactElement => {
+    let coreSystemRule: PlatformComponentRule | undefined;
+    let redHatLayeredProductsRule: PlatformComponentRule | undefined;
+    const customRules: PlatformComponentRule[] = [];
+
+    platformComponentsConfig.rules.forEach((rule) => {
+        if (rule.name === 'system rule') {
+            coreSystemRule = rule;
+        } else if (rule.name === 'red hat layered products') {
+            redHatLayeredProductsRule = rule;
+        } else {
+            customRules.push(rule);
+        }
+    });
+
+    return (
+        <Grid hasGutter>
+            <GridItem sm={12} md={6} lg={4}>
+                <Card isFlat>
+                    <CardTitle>Core system</CardTitle>
+                    <CardBody>
+                        <Stack hasGutter>
+                            <Text>
+                                Components found in core Openshift and Kubernetes namespaces are
+                                included in the platform definition by default.
+                            </Text>
+                            <Divider component="div" />
+                            <Text component="small" className="pf-v5-u-color-200">
+                                Namespaces match (Regex)
+                            </Text>
+                            <CodeBlock>{coreSystemRule?.namespaceRule.regex}</CodeBlock>
+                        </Stack>
+                    </CardBody>
+                </Card>
+            </GridItem>
+            <GridItem sm={12} md={6} lg={4}>
+                <Card isFlat>
+                    <CardTitle>Red Hat layered products</CardTitle>
+                    <CardBody>
+                        <Stack hasGutter>
+                            <Text>
+                                Components found in Red Hat layered and partner product namespaces
+                                are included in the platform definition by default.
+                            </Text>
+                            <Divider component="div" />
+                            <Text component="small" className="pf-v5-u-color-200">
+                                Namespaces match (Regex)
+                            </Text>
+                            <CodeBlock>
+                                <div className="truncate-multiline">
+                                    {redHatLayeredProductsRule?.namespaceRule.regex}
+                                </div>
+                            </CodeBlock>
+                            <StackItem className="pf-v5-u-text-align-center pf-v5-u-mt-sm">
+                                <Button variant="link" isInline>
+                                    View more
+                                </Button>
```

Comment:
For this, I decided to _always_ show the `View more` button. Otherwise, we'd have to check that we're showing an ellipsis and display it. I did a quick check, and it seems like it'll be a bit hacky. We do have a `react-truncate` package, used in only one place, that we could potentially use here.